ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -341,16 +341,23 @@ private void broadcastBlock(final Block block, final BlockHeader parent) {
                     block.getHash());
               } else {
                 final double timeInS = importTask.getTaskTimeInSec();
+                final String blockHash = block.getHash().toHexString();
+                final String shortHash =
+                    String.format(
```

Comment:
My concern with short hashes is then we cannot go to etherscan from the log.  I do this fairly frequently when analyzing short lived forks.

Perhaps put the peers not at the end and making the other numbers fixed width?

Reply:
What about printing out an additional debug level log info with the full hash? My impression is that the full details are too much on info log level.